catalog/app/containers/Bucket/Queries/Queries.tsx

```diff
@@ -167,18 +166,23 @@ export default function Queries({
   const isButtonDisabled = (
     queryContent: requests.ElasticSearchQuery,
     resultsData: requests.AsyncData<requests.ElasticSearchResults>,
+    error: Error | null,
   ): boolean =>
-    !queryContent || !!resultsData.case({ _: R.T, Init: R.F, Err: R.F, Ok: R.F })
+    !!error ||
+    !queryContent ||
+    !!resultsData.case({ _: R.T, Init: R.F, Err: R.F, Ok: R.F })
```

Comment:
```suggestion
    !!resultsData.case({ Pending: R.T, _: R.F })
```

alternative: `AsyncResult.Pending.is(resultsData.result)`